node/src/reactor.rs

```diff
@@ -234,10 +237,42 @@ impl RunnerMetrics {
     /// Create and register new runner metrics.
     fn new(registry: &Registry) -> Result<Self, prometheus::Error> {
         let events = IntCounter::new("runner_events", "total event count")?;
+
+        // Create an event dispatch histogram, putting extra emphasis on the area between 1-10 us.
+        let event_dispatch_duration = Histogram::with_opts(
+            HistogramOpts::new(
+                "event_dispatch_duration",
+                "duration of complete dispatch of a single event in nanoseconds",
+            )
+            .buckets(vec![
+                100.0,
+                500.0,
+                1_000.0,
+                5_000.0,
+                10_000.0,
+                20_000.0,
+                50_000.0,
+                100_000.0,
+                200_000.0,
+                300_000.0,
+                400_000.0,
+                500_000.0,
+                600_000.0,
+                700_000.0,
+                800_000.0,
+                900_000.0,
+                1_000_000.0,
+                2_000_000.0,
+                5_000_000.0,
```

Comment:
Scala lib comes with some default buckets that can be overwritten with config. Are there default buckets for Rust lib but they're too big for us?

Reply:
That sounds useful for us, but confusing for end users faced with an increasingly complex config file?

Maybe we need a `dev`, `debug` or `advanced` section of the config if we do take this approach?

Reply:
In Scala there was a separate file for this (`application.conf`). Node configuration was in its own file and chainspec in its own dir structure. It wasn't convention invented by us but rather used by the library authors (and not specifically Prometheus library but whole Java/Scala ecosystem).

Reply:
Buckets can be configured, they are just an a vec of values passed in. I'd rather have some sensible defaults first though, and make them configurable when we need to.